src/helpers/ens.ts

```diff
@@ -483,6 +485,11 @@ const formatRentPrice = (
   };
 };
 
+const accentColorAtom = atom({
```

Comment:
This is global state so `ENSAssignRecordsBottomActions` can use the accent color set within `ENSAssignRecordsSheet`.